aws/handlers/appointments_api_req_handler.go

```diff
@@ -12,21 +13,29 @@ import (
 )
 
 // HandleAppointmentsRequest will handle all appointment gate api requests
-func HandleAppointmentsRequest(req events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+func HandleAppointments1Request(req events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
 
 	switch req.HTTPMethod {
 	case "GET":
 		db, err := database.CreateDB()
-
+		log.Printf("GET: %s", req.HTTPMethod)
 		if err != nil {
 			return utils.GatewayResponse(http.StatusBadRequest, models.ErrorBody{ErrorMsg: aws.String(err.Error())})
 		}
 
 		repository := repositories.AppointmentsRepository{DB: db}
 		//TO-DO this is work in progress need to add api to get appointments per user/counselor/timeslot
-		appointments, err := repository.FetchAll(nil)
+		counselorId  := req.QueryStringParameters["counselor_id"]
```

Comment:
We should first get the counselor with the given Id, before getting all his appointments...